app/src/main/java/org/oppia/android/app/home/recentlyplayed/RecentlyPlayedActivityPresenter.kt

```diff
@@ -3,17 +3,20 @@ package org.oppia.android.app.home.recentlyplayed
 import androidx.appcompat.app.AppCompatActivity
 import org.oppia.android.R
 import org.oppia.android.app.activity.ActivityScope
+import org.oppia.android.app.model.RecentlyPlayedActivityParams
 import javax.inject.Inject
+import org.oppia.android.app.model.RecentlyPlayedActivityTitle
 
 /** The presenter for [RecentlyPlayedActivity]. */
 @ActivityScope
 class RecentlyPlayedActivityPresenter @Inject constructor(private val activity: AppCompatActivity) {
-  fun handleOnCreate(internalProfileId: Int) {
+  fun handleOnCreate(recentlyPlayedActivityParams: RecentlyPlayedActivityParams) {
+    activity.title = getTitle(recentlyPlayedActivityParams)
```

Comment:
Can you record and share the screen-reader output starting from home screen with both the cases : `Recently Played Stories` and `Stories For You`